checker/tests/nullness/InvariantTypes.java

```diff
@@ -18,6 +18,7 @@ void testAsLocal() {
     }
 
     // Type for array creation is propagated from LHS
+    @SuppressWarnings("invalid.polymorphic.qualifier.use")
     @PolyNull Object[] po = new Object[5];
```

Comment:
This test file is unrelated to my changes. It just uses a polymorphic qualifier on a field.

Reply:
And the polymorphic qualifier is now forbidden. Instead of suppressing this error, two valid tests that actually test what was intended to be tested would be more useful. Or keep this test and add the intended tests.